agent/sshd/server.go

```diff
@@ -122,13 +123,21 @@ func (s *Server) sessionHandler(session sshserver.Session) {
 
 		os.Chown(pts.Name(), uid, -1)
 
+		remoteAddr := session.RemoteAddr().String()
+
 		logrus.WithFields(logrus.Fields{
 			"user":       session.User(),
 			"pty":        pts.Name(),
-			"remoteaddr": session.RemoteAddr().String(),
+			"remoteaddr": remoteAddr,
 			"localaddr":  session.LocalAddr().String(),
 		}).Info("Session started")
 
+		ut := utmpStartSession(
+			strings.TrimPrefix(pts.Name(),"/dev/"),
```

Comment:
I belive the trimming of the prefix is an implementation detail and belongs to the `utmpStartSession(...)` code. So here we pass the `pty` and be done with that.

Reply:
Done.

Reply:
I think you did not push the changes. I assumed it'd be replaced by `pts.Name()` and inside the function the prefix would be trimmed.

Reply:
Now done and pushed.